test/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.test.ts

```diff
@@ -1220,7 +1245,7 @@ describe('VideoPriorityBasedPolicy', () => {
       resub = policy.wantsResubscribe();
       expect(resub).to.equal(false);
 
-      incrementTime(5100);
+      incrementTime(8100);
```

Comment:
I recommend keeping existing tests to ensure that your change is backward compatible.

Reply:
Worth noting since release we have considered small behavior modifications to the priority policy to be acceptable since we label that behavior may change. But I do think that when you have to adjust an existing test we should leave a PR comment explaining why it needed to change :).

Reply:
I see. Since we are changing from comparing to the last subscribe timestamp to comparing a reference timestamp, the behavior will be changed. Other than the description in this PR, do we need to state the change somewhere?

Reply:
In PR is fine as its specific to the diff. e.g. my PRs almost always have 1+ self comments on them immediately on post :).